### DISCUSSION TITLE
In network game tag "ignore" not applies to stacks

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15074] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When you give command  "ignore" to stack o items, the order not works as expected.
Bots continue to see and use these items.

### Reproduction steps

It ONLY manifests in the NETWORK GAME!
I just tested and reveal, this bug is not fixed in the latest version.
To reproduce bug:
Put a stack of items in the closet.
Give the command "ignore" to this stack.
Then drop it on the floor.
Tell the bot to "clean up"
You will see that the bot will take this stack and put it back in the closet.
That is, the command "ignore" did not work on the stack.
For the purity of the experiment, take another stack.
Now scatter this stack one by one.
You can scatter it in a cabinet, or in your inventory, no matter.
Give the command "ignore" to each item.
Now combine them into a stack.
Drop this stack on the floor.
Tell the bot to "clean up"
The bot will ignore this stack as expected.
The only way to guarantee "ignore" items in network game is to give the "ignore" command to every item in the stack.
A simpler method is to deny bots access to a certain cabinet
and keep the meds in that cabinet.
This method works as expected.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14427
